docs/source/user_guide/dataset_creation/datasets.rst

```diff
@@ -2666,6 +2669,164 @@ as follows:
+    ``dataset_dir``.
+
+    Currently, only single frame images are supported, but a community
+    contribution to support 3D or 4D image types (e.g., CT scans) is welcomed!
+
+.. note::
+
+    See :class:`DICOMDatasetImporter <fiftyone.utils.dicom.DICOMDatasetImporter>`
+    for parameters that can be passed to methods like
+    :meth:`Dataset.from_dir() <fiftyone.core.dataset.Dataset.from_dir>` to
+    customize the import of datasets of this type.
+
+You can create a FiftyOne dataset from a DICOM dataset stored in standard
+format as follows:
+
+.. tabs::
+
+  .. group-tab:: Python
+
+    .. code-block:: python
+        :linenos:
+
+        import fiftyone as fo
+
+        name = "my-dataset"
+        dataset_dir = "/path/to/dicom-dataset"
+
+        # Create the dataset
+        dataset = fo.Dataset.from_dir(
+            dataset_dir=dataset_dir,
+            dataset_type=fo.types.DICOMDataset,
+            name=name,
+        )
+
+        # View summary info about the dataset
+        print(dataset)
+
+        # Print the first few samples in the dataset
+        print(dataset.head())
+
+  .. group-tab:: CLI
+
+    .. code-block:: shell
+
+        NAME=my-dataset
+        DATASET_DIR=/path/to/dicom-dataset
+
+        # Create the dataset
+        fiftyone datasets create \
+            --name $NAME \
+            --dataset-dir $DATASET_DIR \
+            --type fiftyone.types.DICOMDataset
+
+        # View summary info about the dataset
+        fiftyone datasets info $NAME
+
+        # Print the first few samples in the dataset
+        fiftyone datasets head $NAME
+
+You can create a FiftyOne dataset from a glob pattern of DICOM files or the
+path to a DICOMDIR file as follows:
+
+.. tabs::
+
+  .. group-tab:: Python
+
+    .. code-block:: python
+        :linenos:
+
+        import fiftyone as fo
+
+        name = "my-dataset"
+
+        dicom_path = "/path/to/*.dcm"  # glob pattern of DICOM files
+        # dicom_path = "/path/to/DICOMDIR"  # DICOMDIR file
+
+        # Create the dataset
+        dataset = fo.Dataset.from_dir(
+            dicom_path=dicom_path,
+            dataset_type=fo.types.DICOMDataset,
+            keywords=["PatientName", "StudyID"],  # load specific attributes
+            name=name,
+        )
+
+        # View summary info about the dataset
+        print(dataset)
+
+        # Print the first few samples in the dataset
+        print(dataset.head())
+
+  .. group-tab:: CLI
+
+    .. code-block:: shell
+
+        NAME=my-dataset
+
+        DICOM_PATH='/path/to/*.dcm'  # glob pattern of DICOM files
+        # DICOM_PATH='/path/to/DICOMDIR'  # DICOMDIR file
+
+        # Create the dataset
+        fiftyone datasets create \
+            --name $NAME \
+            --type fiftyone.types.DICOMDataset \
+            --kwargs \
+                dicom_path=$DICOM_PATH \
+                keywords=PatientName,StudyID  # load specific attributes
```

Comment:
```suggestion
                label_field="" \
                keywords=PatientName,StudyID  # load specific attributes
```

Reply:
Yeah, I'm actually planning a more general update to how `label_field` is used so that `ground_truth` won't be prepended when there are >1 fields being imported.

So I'll punt on this change for now because after the aforementioned change the current syntax will do the right thing.